php/php.composer/src/org/netbeans/modules/php/composer/files/ComposerJson.java

```diff
@@ -75,6 +78,14 @@ public File getVendorDir() {
         return new File(getFile().getParentFile(), vendorDir);
     }
 
+    public Set<String> getScripts() {
+        Map<String, Object> scripts = composerJson.getContentValue(Map.class, FIELD_SCRIPTS);
+        if (scripts == null) {
+            return Collections.emptySet();
+        }
+        return scripts.keySet();
```

Comment:
This means that the order of the scripts is undefined - is this expected? I would likely expect (a) order as it is in the file or maybe better (b) alphabetical sort.

Reply:
Actually, not sure what Map implementation is used by the JSON parser so it could be linked hashMap. Could you pleae verify it? Thank you.

Reply:
Aha, now I see that the sorting is in the action itself.